KarhooUISDK/CommonUI/AddressView/AddressView.swift

```diff
@@ -15,6 +15,15 @@ struct KarhooAddressView: View {
 
     // MARK: - Nested types
 
+    private enum Constants {
+        static var maxWidth: CGFloat { UIScreen.main.bounds.width - UIConstants.Spacing.standard * 2 }
+        static let dotsColumnWidth: CGFloat = 10
+        static let labelHeight: CGFloat = 32
+        static let padding: CGFloat = 10
+        static let roundIconSide: CGFloat = 10
+        static let minimumScaleFactor: CGFloat = 0.7
+    }
```

Comment:
Why you didn't use values from UIConstans?

Reply:
We have spacing, corner radius, test size etc.

Reply:
I've discussed this with Aleks when reviewing his PR for the new address component and he said that the UI didn't look ok with the pre-existing values. He got the best results with these values. Since they are meant to be used locally, adding them to the general constants enum didn't make sense.